WordPress/Classes/Models/Post.m

```diff
@@ -196,18 +196,19 @@ - (NSString *)contentPreviewForDisplay
 
 - (NSString *)statusForDisplay
 {
-    NSString *statusString = [self statusTitle];
-    if ([self.status isEqualToString:PostStatusPublish] || [self.status isEqualToString:PostStatusDraft]) {
-        statusString = nil;
-    }
-    if (![self hasRemote]) {
+    NSString *statusString = nil;
```

Comment:
Fwiw, the nil assignment is unnecessary and the declared property resolves to nil with out it.